{"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError"},"kind":"symbol","sections":[],"topicSections":[{"identifiers":["doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidCallback","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidIdToken"],"title":"Enumeration Cases"},{"identifiers":["doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/Equatable-Implementations","doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/Error-Implementations"],"generated":true,"title":"Default Implementations"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"tokens":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"SignInWithAppleError","kind":"identifier"}],"languages":["swift"]}]}],"hierarchy":{"paths":[["doc:\/\/RxFireAuth\/documentation\/RxFireAuth"]]},"metadata":{"role":"symbol","modules":[{"name":"RxFireAuth"}],"navigatorTitle":[{"kind":"identifier","text":"SignInWithAppleError"}],"symbolKind":"enum","fragments":[{"kind":"keyword","text":"enum"},{"text":" ","kind":"text"},{"text":"SignInWithAppleError","kind":"identifier"}],"externalID":"s:10RxFireAuth20SignInWithAppleErrorO","roleHeading":"Enumeration","title":"SignInWithAppleError"},"schemaVersion":{"minor":3,"patch":0,"major":0},"abstract":[{"type":"text","text":"Errors thrown by "},{"code":"SignInWithAppleHandler","type":"codeVoice"},{"type":"text","text":" instances."}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/rxfireauth\/signinwithappleerror"]}],"relationshipsSections":[{"title":"Conforms To","kind":"relationships","identifiers":["doc:\/\/RxFireAuth\/SQ","doc:\/\/RxFireAuth\/s5ErrorP","doc:\/\/RxFireAuth\/SH","doc:\/\/RxFireAuth\/s8SendableP"],"type":"conformsTo"}],"references":{"doc://RxFireAuth/s5ErrorP":{"type":"unresolvable","identifier":"doc:\/\/RxFireAuth\/s5ErrorP","title":"Swift.Error"},"doc://RxFireAuth/SH":{"type":"unresolvable","identifier":"doc:\/\/RxFireAuth\/SH","title":"Swift.Hashable"},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError/Equatable-Implementations":{"url":"\/documentation\/rxfireauth\/signinwithappleerror\/equatable-implementations","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/Equatable-Implementations","abstract":[],"title":"Equatable Implementations","type":"topic","kind":"article","role":"collectionGroup"},"doc://RxFireAuth/s8SendableP":{"type":"unresolvable","identifier":"doc:\/\/RxFireAuth\/s8SendableP","title":"Swift.Sendable"},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError/Error-Implementations":{"role":"collectionGroup","url":"\/documentation\/rxfireauth\/signinwithappleerror\/error-implementations","type":"topic","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/Error-Implementations","kind":"article","abstract":[],"title":"Error Implementations"},"doc://RxFireAuth/SQ":{"type":"unresolvable","identifier":"doc:\/\/RxFireAuth\/SQ","title":"Swift.Equatable"},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError/invalidIdToken":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidIdToken","role":"symbol","abstract":[{"type":"text","text":"Apple returned an invalid ID token."}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidIdToken"}],"type":"topic","title":"SignInWithAppleError.invalidIdToken","url":"\/documentation\/rxfireauth\/signinwithappleerror\/invalididtoken","kind":"symbol"},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError/invalidCallback":{"identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError\/invalidCallback","url":"\/documentation\/rxfireauth\/signinwithappleerror\/invalidcallback","role":"symbol","type":"topic","kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"invalidCallback","kind":"identifier"}],"title":"SignInWithAppleError.invalidCallback","abstract":[{"type":"text","text":"A callback has been invoked without a nonce."}]},"doc://RxFireAuth/documentation/RxFireAuth/SignInWithAppleError":{"kind":"symbol","title":"SignInWithAppleError","abstract":[{"type":"text","text":"Errors thrown by "},{"code":"SignInWithAppleHandler","type":"codeVoice"},{"type":"text","text":" instances."}],"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"SignInWithAppleError"}],"url":"\/documentation\/rxfireauth\/signinwithappleerror","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth\/SignInWithAppleError","type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"SignInWithAppleError","kind":"identifier"}]},"doc://RxFireAuth/documentation/RxFireAuth":{"url":"\/documentation\/rxfireauth","kind":"symbol","identifier":"doc:\/\/RxFireAuth\/documentation\/RxFireAuth","role":"collection","title":"RxFireAuth","type":"topic","abstract":[]}}}